### DISCUSSION TITLE
black square over face at live feed

### DISCUSSION THREAD

**Discussion by K4ndiszucker** [hacksider/Deep-Live-Cam#300] [Q&A] [answered]
over my face is a black square when changing face live. tried different starting pictures. also python doesnt show any error and the video is very very laggy
![Bild_2024-08-13_215356059](https://github.com/user-attachments/assets/49244879-482d-4d9b-8f10-429cf84bd090)

**Comment by Ku-Tadao** [comment]
https://github.com/hacksider/Deep-Live-Cam/discussions/242

**Reply by K4ndiszucker** [reply]
![Bild_2024-08-13_224710932](https://github.com/user-attachments/assets/2be326d2-a835-4bf2-b990-c795d6c92af1)

**Comment by davidharket** [comment]
Getting the same error, and i am running on 3.10.00

**Comment by Ku-Tadao** [comment]
@K4ndiszucker @davidharket what are the args both of you use when running the py? 

if it's Cuda i can maybe assist a tiny bit, otherwise i'm not really sure 😅

**Comment by FilipeF12** [comment, marked as answer]
Had BlackBox face with CPU.
Had PizzaFace with GPU.

Fixed both by replacing [inswapper_128_fp16.onnx] that HuggingFace spit out... (size 270,790) with newer one I had from a different project
(size 541,263).

The one linked in the install instructions did **NOT** work. 

This one worked ---v

https://huggingface.co/Devia/G/blob/main/inswapper_128.onnx

Rename to [ inswapper_128_fp16.onnx ] looks like that name hard coded.

![strio](https://github.com/user-attachments/assets/8bc0f216-ac4e-4766-8327-d3a2a2f7e318)

**Comment by caleb-friesen** [comment]
@FilipeF12 This worked for me with CPU in Live mode, thank you!
![elonmusk](https://github.com/user-attachments/assets/35b6d521-83fa-4057-a7b2-4baf6be6fbe1)
